template.html

```diff
@@ -54,13 +54,19 @@ <h3 class="text-muted">fedmsg health</h3>
   </div></div>
   <div class="row">
     <div class="col-md-4">
-      <img class="graph center-block" src="https://admin.fedoraproject.org/collectd/bin/graph.cgi?hostname=${params['host']};plugin=${params['plugin']};plugin_instance=${params['instance']};type=gauge;type_instance=${params['consumer']}_in;begin=-${span}">
+      <a target="_blank" href="https://admin.fedoraproject.org/collectd/bin/graph.cgi?hostname=${params['host']};plugin=${params['plugin']};plugin_instance=${params['instance']};type=gauge;type_instance=${params['consumer']}_out;begin=-${span}">
```

Comment:
Looks like a typo on line 57 here.  it has ``_out`` on the end where it should have ``_in``.  Will merge after you fix.